src/scripts/migrate.py

```diff
@@ -0,0 +1,142 @@
+"""Slurp migration pipeline
+
+TODO's:
+  -
+"""
+import os.path
+from argparse import ArgumentParser
+from typing import Dict, List
+
+import pandas as pd
+import oaklib
+from oaklib.implementations import ProntoImplementation
+from oaklib.resource import OntologyResource
+
+
+# TODO: implement this func:
+# todo: IDs should be int or str? prolly str
+def determine_next_available_mondo_id(min_id: str, mondo_termlist_df: pd.DataFrame) -> str:
+    """Starting from `min_id`, count up and check until finding the next ID."""
+    next_id = str(0)
+    return next_id
+
+
+def _get_direct_parents(curie: str, ontology: ProntoImplementation):
+    """Get CURIEs of parents of a class
+    todo: `ontology` typing: should probably be typed agnostic of the implementation. But it inherits from so
+     much stuff, I'm not sure which if any of these to use. I was expecting `BasicOntologyInterface`, but it doesn't
+     inherit from that: RelationGraphInterface, OboGraphInterface, ValidatorInterface, SearchInterface,
+     SubsetterInterface, MappingProviderInterface, PatcherInterface, SemanticSimilarityInterface, MetadataInterface,
+     DifferInterface, ABC
+    """
+    # TODO: How to get just the *direct* parents? (a) func for that? (b) look at relationships and see if subClass?
+    # rels = ontology.get_outgoing_relationships_by_curie(curie)  # works for ProntoImplementation
+    rels = ontology.relationships([curie])
+    p = []
+    for rel, parents in rels.items():
+        # print(f'  {rel} ! {ontology.get_label_by_curie(rel)}')
+        for parent in parents:
+            # print(f'    {parent} ! {ontology.get_label_by_curie(parent)}')
+            p.append(parent)
+    return p
+
+
+def _get_all_term_curies(ontology: ProntoImplementation, is_omim=False) -> List[str]:
```

Comment:
Had some issues getting terms to show when running from makefile: #56

Reply:
did you sort this?

Reply:
Ah yes, I'm pretty sure this is fixed. If not, I'll bring this up again later.